src/cff.h

```diff
@@ -25,7 +25,7 @@ struct CFFIndex {
 class OpenTypeCFF : public Table {
  public:
   explicit OpenTypeCFF(Font *font, uint32_t tag)
-      : Table(font, tag),
+      : Table(font, tag, OTS_TAG_CFF),
```

Comment:
Can’t we just use `tag` here in place of `OTS_TAG_CFF` since they should be the same? Ditto in other places where we don’t pass 0 for the type.